ui/page/show/view.jsx

```diff
@@ -84,7 +84,12 @@ function ShowPage(props: Props) {
       // Only redirect if we are in lbry.tv land
       // replaceState will fail if on a different domain (like webcache.googleusercontent.com)
       if (canonicalUrlPath !== window.location.pathname && DOMAIN === window.location.hostname) {
-        history.replaceState(history.state, '', canonicalUrlPath);
+        const urlParams = new URLSearchParams(search);
```

Comment:
A comment here explaining the value of this constant would help `COLLECTIONS_CONSTS.COLLECTION_ID`